docs-src/clusters/quick-install.md

```diff
@@ -1,85 +1,47 @@
 
-### Temporalite
+### Temporal CLI
 
-Temporalite is a distribution of Temporal that runs as a single process with zero runtime dependencies.
+Temporal CLI is a distribution of Temporal that runs as a single process with zero runtime dependencies.
 It supports persistence to disk and in-memory mode through SQLite.
 
-**Prerequisites**
-
-Temporalite requires [Go 1.18 or later](https://go.dev/dl/).
-
-**Build and start Temporalite**
-
-The following steps start and run a Temporal Cluster.
-
-1. Build from source.
-   ```bash
-   git clone https://github.com/temporalio/temporalite.git
-   cd temporalite
-   go build ./cmd/temporalite
-   ```
-2. Start Temporalite by using the `start` command.
-   ```bash
-   ./temporalite start --namespace default
-   ```
-   Replace `default` with your [Namespace Name](/namespaces).
-
-**Results**: You should have Temporal Cluster running at `http://127.0.0.1:7233` and the Temporal Web UI at [`http://127.0.0.1:8233`](http://127.0.0.1:8233/).
+**Install and start the Temporal CLI**
 
-<!-- For macOS users, if you receive the `error setting up schema: stat /Users/<user_name>/Library/Application Support/temporalite/db:` error, then create the folders `temporalite/db` in your `Application Support` library. -->
+1. Choose one of the following install methods.
+   1. Install the Temporal CLI with Homebrew.
 
-### Docker Compose
+      ```bash
+      brew install temporal
+      ```
 
-Use Docker Compose and Temporal Cluster Docker images to quickly install and run a Temporal Cluster locally while developing Temporal Applications.
+   2. Install the Temporal CLI with cURL.
 
-**Prerequisites**
+      ```bash
+      curl -sSf https://temporal.download/cli.sh | sh
+      ```
 
-Install [Docker](https://docs.docker.com/engine/install) and [Docker Compose](https://docs.docker.com/compose/install).
+   3. Install the Temporal CLI for Windows.
+      1. Download and run the installer for the [latest release](https://github.com/temporalio/cli/releases/latest/).
+      2. Add the `temporal.exe` binary to your PATH.
+2. Start Temporal Server by using the `start-dev` command.
 
-**Clone the repo and run Docker Compose**
-
-The following steps start and run a Temporal Cluster using the default configuration.
-
-1. Clone the [temporalio/docker-compose](https://github.com/temporalio/docker-compose) repository.
-   ```bash
-   git clone https://github.com/temporalio/docker-compose.git
-   ```
-2. Change to the directory for the project.
-   ```bash
-   cd docker-compose
-   ```
-3. From your project directory, start your application.
    ```bash
-   docker compose up
+   temporal server start-dev
    ```
 
-**Results**: You should have Temporal Cluster running at `http://127.0.0.1:7233` and the Temporal Web UI at [`http://127.0.0.1:8080`](http://127.0.0.1:8080/).
+To customize the [Namespace Name](/namespaces), start the server with the `--namespace` command.
```

Comment:
You're missing a sentence here mentioning that by default the "default" namespace is created, otherwise this comes out of context.

You can copy some more useful information from the cli's README: https://github.com/temporalio/cli/